src/ucp/tag/eager_snd.c

```diff
@@ -214,11 +232,28 @@ static ucs_status_t ucp_tag_eager_sync_bcopy_single(uct_pending_req_t *self)
 
 static ucs_status_t ucp_tag_eager_sync_bcopy_multi(uct_pending_req_t *self)
 {
-    ucs_status_t status = ucp_do_am_bcopy_multi(self,
-                                                UCP_AM_ID_EAGER_SYNC_FIRST,
-                                                UCP_AM_ID_EAGER_MIDDLE,
-                                                ucp_tag_pack_eager_sync_first_dt,
-                                                ucp_tag_pack_eager_middle_dt, 1);
+    ucp_request_t *req = ucs_container_of(self, ucp_request_t, send.uct);
+    size_t length;
+    ucs_status_t status;
+
+    if (ucs_unlikely(ucp_wireup_ep_test(req->send.ep->uct_eps[req->send.lane]))) {
+        return UCS_ERR_NO_RESOURCE;
+    }
```

Comment:
isn't it handled by `ucp_wireup_ep_am_bcopy` already?

Reply:
yes, but it's too late